src/current/v24.3/known-limitations.md

```diff
@@ -12,14 +12,14 @@ docs_area: releases
 
 This section describes newly identified limitations in CockroachDB {{ page.version.version }}.
 
+### Generic query plans
+
+{% include {{ page.version.version }}/known-limitations/generic-query-plan-limitations.md %}
```

Comment:
I noticed that v24.2 also has this include file: `known-limitations/generic-query-plan-limitations.md`. However, v24.2 `known-limitations.md` does not have this include reference. Should v24.2 have this include reference or should the include file be deleted?

Reply:
This is a great catch. Technically, the limitation existed in 24.2. However, they weren't identified until after 24.2. I'm going to retroactively put them under "New in 24.2" because that is when they first existed. This would be a good topic to discuss with the team.